deploy/helm/magda/charts/opa/policies/object/registry/record/groups.rego

```diff
@@ -0,0 +1,5 @@
+package object.registry.record
```

Comment:
Can we name this `esri-groups.rego` or something like that so it's obvious that it's going to use the `esri-access-groups` aspect? :)

Reply:
done